setup.py

```diff
@@ -103,8 +105,13 @@ def get_library_names(self, build_type):
     else:
       return ndt + nd
 
-target = Target(DIST_TARGET)
+  def get_requirements(self):
+    reqs = open('dev-requirements.txt').read().strip().split('\n')
+    if self.target != 'ndt':
+      reqs.append('dynd.ndt')
```

Comment:
This solution is apparently not possible:

https://travis-ci.org/libdynd/dynd-python/jobs/134413087#L4138

I thought that conda would ignore dependencies in setup.py.